docs/csharp/index.yml

```
@@ -14,57 +14,113 @@ landingContent:
+      - linkListType: tutorial
         links:
-          - text: Download the .NET SDK
-            url: https://dotnet.microsoft.com/download
+          - text: "Self-guided tutorials"
+            url: https://docs.microsoft.com/en-us/users/dotnet/collections/yz26f8y64n7k07?WT.mc_id=dotnet-35129-website 
+          - text: "In-browser tutorial"
+            url: ./tour-of-csharp/tutorials/hello-world.yml
 
   - title: "C# fundamentals"
     linkLists:
       - linkListType: overview
         links:
           - text: "A tour of C#"
             url: tour-of-csharp/index.md
-          - text: "Structure of a C# program"
+          - text: "Inside a C# program"
             url: ./fundamentals/program-structure/index.md
-          - text: Main() and command-line arguments
-            url: ./fundamentals/program-structure/main-command-line.md
+          - text: "C# highlights video series"
+            url: https://youtube.com/playlist?list=PLdo4fOcmZ0oU3ZLx6Ul1_HPrr6lFPIn9O
       - linkListType: concept
         links:
-          - text: "The C# and .NET type system"
-            url: fundamentals/types/index.md
-          - text: Language Integrated Query (LINQ)
-            url: linq/index.md
-          - text: async and await
-            url: programming-guide/concepts/async/index.md
+          - text: "Type system"
+            url: ./fundamentals/types/index.md
+          - text: "Object oriented programming"
+            url: ./fundamentals/object-oriented/index.md
+          - text: "Functional techniques"
+            url: ./fundamentals/functional/pattern-matching.md
+          - text: "Exceptions"
+            url: ./fundamentals/exceptions/index.md
+          - text: "Coding style"
+            url: ./fundamentals/coding-style/identifier-names.md
       - linkListType: tutorial
         links:
+          - text: Display command-line
+            url: ./fundamentals/tutorials/how-to-display-command-line-arguments.md
+          - text: Intro to classes
+            url: ./fundamentals/tutorials/classes.md
+          - text: Object oriented C#
+            url: ./fundamentals/tutorials/oop.md
           - text: Object-oriented inheritance
             url: ./fundamentals/object-oriented/inheritance.md
+          - text: Converting types
+            url: ./fundamentals/tutorials/safely-cast-using-pattern-matching-is-and-as-operators.md
+          - text: Pattern matching
+            url: ./fundamentals/tutorials/pattern-matching.md
+          - text: Exception handling
+            url: ./fundamentals/exceptions/index.md
           - text: Use LINQ to query data
             url: tutorials/working-with-linq.md
-          - text: Format text using string interpolation
-            url: tutorials/string-interpolation.md
-          - text: Browse all tutorials
-            url: ./fundamentals/tutorials/classes.md
-  
-  - title: "New features in C# 10"
+  - title: "Key Concepts"
```

nit: use sentence capitalization:

```suggestion
  - title: "Key concepts"
```